.vsconfig

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
What is this file used for?

Reply:
VS will now check for the workloads (ie validation tools) on startup and offer to install it if you don't have it.